From 1e6dd8a74125e78dc87521fb2cd161719600092a Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Fri, 19 Jan 2024 06:56:43 +0100
Subject: [PATCH] com.webos.notification.perm.json: Fix incorrect values

We don't have tv.settings and we need application.launcher for call to SAM/launch.

Fixes:

WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'tv.settings' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.notification.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'applicationmanager.query' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.notification.perm.json

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 files/sysbus/com.webos.notification.perm.json.in | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/files/sysbus/com.webos.notification.perm.json.in b/files/sysbus/com.webos.notification.perm.json.in
index 47f7bec..c8d78ae 100644
--- a/files/sysbus/com.webos.notification.perm.json.in
+++ b/files/sysbus/com.webos.notification.perm.json.in
@@ -1,10 +1,9 @@
 {
     "com.webos.notification": [
+        "application.launcher",
         "application.operation",
-        "applicationmanager.query",
         "database.operation",
         "settings.query",
-        "time.query",
-        "tv.settings"
+        "time.query"
     ]
 }
